### ISSUE TITLE
Can't retrieve status/options/upf with API 3.11

### ISSUE THREAD

**Issue by pjamar** [florianl/panonoctl#5]
getStatus, getOptions and getUpfs() do not have their expected returns on API 3.11.

See the following session:
```Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import panonoctl
>>> cam = panonoctl.panono()
>>> cam.connect()
True
>>> cam.auth()
{"jsonrpc": "2.0", "params": {"device": "test", "force": "test"}, "id": 1, "method": "auth"}
{'jsonrpc': '2.0', 'id': 1, 'result': {'battery_value': 79, 'current_time': '2017-01-01 10:08:42,000', 'is_auth': True, 'capture_available': True, 'power_charging_status': 'disconnected', 'sound_config': 'normal', 'update_ready': True, 'charging_status': 'not_charging', 'auth_token': 'XXXXXXXXXXXXXXXXXXXXX', 'storage': {'panorama': {'usage': 2169860096, 'total': 13597483008, 'capacity_available': True}, 'cache': {'usage': 70258688, 'total': 1323102208, 'capacity_available': True}}, 'log_serialize_ready': True, 'accessory': [], 'firmware_update_url': 'http://192.168.80.80:80/update_image', 'power_status': 'not_connected', 'serial_number': 'DEJASN', 'firmware_version': '3.2-Dandelion.product.749', 'usb_config': 'mtp', 'api_version': '3.11', 'device_id': 'XXXXXX'}}
>>> cam.getStatus()
{"jsonrpc": "2.0", "id": 2, "method": "get_status"}
>>> cam.getOptions()
{"jsonrpc": "2.0", "id": 3, "method": "get_options"}
>>> cam.getUpfs()
{"jsonrpc": "2.0", "id": 4, "method": "get_upf_infos"}
>>> cam.getServer()
'3.2-Dandelion.product.749'
>>> cam.getApiVersion()
'3.11'
```

**Comment by florianl** [comment]
Hi @pjamar 
Thanks for the report. I didn't notice the changes so far. But I will take a closer look at it. 
For the last commit, I ported the code to Python3.

Cheers,
 Florian

**Comment by florianl** [comment]
Hi @pjamar 
On my Panono there is already Server version 3.3.765 with API version 3.12 installed.
At the moment, I can not reproduce your report.

Here is an example output for getOptions:
`{'jsonrpc': '2.0', 'id': 3, 'result': {'AntiShaking': True, 'ExposureTime': 1, 'SaveLocation': True, 'AutoExposure': True, 'ColorTemperature': '0', 'UpperBracketFactor': 4, 'ISO': '100', 'TriggerDelay': 0, 'LowerBracketFactor': 0.125, 'RotationSpeedThresholds': 150, 'ImageType': 'Default'}}
`

**Comment by pjamar** [comment]
Thanks for taking a look at this!
I'll try to reproduce it, update the camera and try again.

**Comment by florianl** [comment]
Hi @pjamar 
I think, I fixed the problem.
The Panono sends periodically status_updates about the battery. These asynchronous updates have not been handeld so far. The recent update https://github.com/florianl/panonoctl/commit/7882d6bf6353c871c655a82ebcc9d913687f7850 fixes this.

Cheers,
 Florian

TL;DR: It works again. Please open new bug report, if not.

**Comment by pjamar** [comment]
Sorry, floriani, I already updated the Panono to 3.3.765, which fixed the issue.
